apps/ledger-live-desktop/tests/fixtures/common.ts

```diff
@@ -35,7 +36,12 @@ type TestFixtures = {
   simulateCamera: string;
 };
 
+const IS_MOCK = process.env.MOCK == "0";
 const IS_DEBUG_MODE = !!process.env.PWDEBUG;
+if (IS_MOCK) {
+  setEnv("SPECULOS_PID_OFFSET", Math.floor(Math.random() * 1000));
```

Comment:
bad idea, 0.1% chance to have collision on each instance spawned
we peer programmed together on an improved version that didn't need this at all